packages/docs/src/routes/docs/(qwik)/components/rendering/index.mdx

```diff
@@ -169,6 +170,8 @@ export default component$(() => {
 })
 ```
 
+> It does not work with `useStore` since it does not return a signal, but you can still use the standard approach as shown below.
```

Comment:
```suggestion
> It does not work with `useStore` since it does not return a signal, but you can still use the manual approach ( value + onInput$ ) as shown below.
```
WDYT?